sbbbattlesim/treasures/__init__.py

```diff
@@ -26,6 +26,18 @@ def pretty_print(self):
     def valid(self):
         return self._level != 0
 
+    def to_state(self):
+        print("TREASURE")
```

Comment:
remove debug prints

Reply:
yeah I think this change is not needed

Reply:
yeah this was not needed and I tested that it worked w/o